provider/marathon/marathon.go

```diff
@@ -369,6 +449,62 @@ func (p *Provider) hasCircuitBreakerLabels(application marathon.Application) boo
 	return ok
 }
 
+// Map of services properties
+// we can get it with label[serviceName][propertyName] and we got the propertyValue
+type servicePropertyValues map[string]string
+
+type labelServiceProperties map[string]servicePropertyValues
+
+// Check if for the given application, we find labels that are defining services
+func (p *Provider) hasServices(application marathon.Application) bool {
```

Comment:
Apologies -- I missed you saying that you already have this very use case. I thought we were still speaking abstractly here.

In that case, by all means keep the function in this PR. My concern was it might take months or years for anyone to come by and request this feature while we had to maintain it. Since you're already around using, let's keep it. :-)

(Sorry for the misunderstanding.)